docs/src/architectures/alchemical-model.rst

```diff
@@ -0,0 +1,46 @@
+.. _architecture-alchemical-model:
+
+Alchemical Model
+================
+
+This is an implementation of Alchemical Model: a Behler-Parrinello neural network
+with SOAP features and Alchemical Compression of the composition space. This model
+is extremely useful for simulating systems with large amount of chemical elements.
+
+For further details, please refer to the original papers:
+
+- Willatt, Michael J., Félix Musil, and Michele Ceriotti. "Feature optimization for
+  atomistic machine learning yields a data-driven construction of the periodic table of
+  the elements." Physical Chemistry Chemical Physics 20.47 (2018): 29661-29668.
+
+- Lopanitsyna, Nataliya, et al. "Modeling high-entropy transition metal alloys with
+  alchemical compression." Physical Review Materials 7.4 (2023): 045802.
+
+- Mazitov, Arslan, et al. "Surface segregation in high-entropy alloys from alchemical
+  machine learning." arXiv preprint arXiv:2310.07604 (2023).
```

Comment:
I'm not sure why this one is in the references =). The other two paper introduce the architecture, but this one mainly uses the same architecture without a change.

Reply:
TBH, given the effort @abmazitov has put into implementing this, I wouldn't find it inappropriate to ask for a citation to his paper, even though strictly speaking it still used a previous implementation.

Reply:
TBD then

Reply:
You can also now cite papers based on a bibtex file located in `docs/static/refs.bib`. Take a look into the `docs/src/architectures/soap-bpnn.rst`. Could be nice to add this if you have time.

Reply:
Thank you for suggestion, @PicoCentauri! Done.